compiler/optimizer/abstractinterpreter/IDTNode.hpp

```diff
@@ -107,6 +107,9 @@ class IDTNode
    float getCallRatio() { return _callRatio; }
    double getRootCallRatio() { return _rootCallRatio; }
 
+   void setFake() { _isFake = true; }
+   bool isFake() { return _isFake; }
```

Comment:
What's `isFake()` for?

Reply:
Sorry, I forgot to remove them. This `_isFake` was introduced when I implemented another version of `copyDescendants()`, which was used to mark the nodes that was interpreted but wasn't added into the IDT. 

Since we no longer use `copyDescendants()`, I removed all `_isFake` related contents now.